src/odemis/driver/test/lakeshore_test.py

```diff
@@ -35,36 +35,82 @@
 TEST_NOHW = (os.environ.get("TEST_NOHW", "0") != "0")  # Default to Hw testing
 
 # arguments used for the creation of basic components
-CONFIG = {"name": "Lakeshore Test",
-          "role": "temperature-controller",
-          "port": "/dev/ttyUSB0",
-          "sensor_input": "B",
-          "output_channel": 2,
+CONFIG_SERIAL_BUS_CONNECTION = {
+    "name": "Lakeshore Test",
+    "role": "temperature-controller",
+    "port": "/dev/ttyUSB0/",
+    # "port": "/dev/fake",
+    "sensor_input": "B",
+    "output_channel": 2,
+}
+
+# arguments used for the creation of basic components
+CONFIG_IP_BUS_CONNECTION = {
```

Comment:
The naming "IP" suggest it's going to test the IP connection, but on the simulator it test the serial port. In addition, it test multiple temperatures. So maybe it'd make more sense to name them "CONFIG_SINGLE_TEMP` and `CONFIG_MULTI_TEMP`, and as a comment indicate it also test the IP connection ?